java/src/test/java/com/saucelabs/saucebindings/VisualTestingTests.java

```diff
@@ -0,0 +1,21 @@
+package com.saucelabs.saucebindings;
+
+import org.junit.Test;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import static org.mockito.Mockito.*;
+
+public class VisualTestingTests {
+    private final SauceOptions sauceOptions = spy(new SauceOptions());
+    JavascriptExecutor dummyJSDriver = mock(RemoteWebDriver.class);
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6386972)